plat/linuxu/include/linuxu/syscall-x86_64.h

```diff
@@ -38,7 +38,7 @@
 
 #define __SC_READ    0
 #define __SC_WRITE   1
-#define __SC_OPEN    2
+#define __SC_OPENAT  257
```

Comment:
Shouldn't open still be retained? And then `__SC_OPENAT` placed in the correct ordering (i.e. after `__SC_CLOCK_GETTIME`)?

Reply:
I didn't see any place where the `__SC_OPEN` macro is used. And because now the open command is implemented with the `openat` syscall I didn't find any reason to keep it. It would be nice to keep only the `openat` system call to keep consistency between all architectures. If you consider that it would be ok to let it there, I will put it back.

Reply:
Ok yes `__SC_OPEN` can be removed.  I still think the order should change -- but looking through other syscall nrs show that no ordering has been kept (at least, in the arm definitions).